lib/beaker/shared/host_manager.rb

```diff
@@ -102,7 +102,7 @@ def run_block_on hosts = [], filter = nil, opts = {}, &block
         end
         if block_hosts.is_a? Array
           if block_hosts.length > 0
-            if opts[:run_in_parallel]
+            if opts[:run_in_parallel] == true
```

Comment:
Can we instead ensure that `opts[:run_in_parellel]` will always be a boolean?  Otherwise we'll have to account for the case when it's a non-boolean value in multiple places, such as any/every other `if` statement.